client/my-sites/pages/page-list.jsx

```diff
@@ -141,49 +175,41 @@ const Pages = localize( React.createClass( {
 			);
 		}
 
-		const { site, siteId } = this.props;
+		const { site, siteId, status = 'published' } = this.props;
 		const sitePart = site && site.slug || siteId;
 		const newPageLink = this.props.siteId ? '/page/' + sitePart : '/page';
 		let attributes;
 
-		if ( this.props.hasRecentError ) {
```

Comment:
Not right now, but I couldn't really repro any scenario leading to this error, and other instances that use `QueryPosts` (or `QueryThemes`, which works quite similarly) don't make use of that sort of error check either (they mostly seem to fall back to a more generic error message like the one above), so I chose to simply drop it, at least for now.